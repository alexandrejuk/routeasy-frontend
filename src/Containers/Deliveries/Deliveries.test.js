import React from 'react'
import { render } from '@testing-library/react'

import Delivery from './'

describe('Delivery component', () => {
  it('should render correctly', () => {
    const callback = jest.fn()

    const { container } = render(
      <Delivery
        data={[]}
        onSubmit={callback}
        actionTable={{}}
        geoCode={callback}
      />
    )

    expect(container).toMatchSnapshot()
  })
})
