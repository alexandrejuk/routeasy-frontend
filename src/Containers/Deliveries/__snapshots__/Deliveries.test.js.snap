// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Delivery component should render correctly 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="sideBar"
    >
      <div
        class="form"
      >
        <div
          class="formGroup"
        >
          <input
            class="input"
            name="name"
            placeholder="Nome Cliente"
            type="text"
            value=""
          />
          <small
            class="error"
          />
        </div>
        <div
          class="formGroup"
        >
          <input
            class="input"
            name="weight"
            placeholder="Peso da Entrega"
            type="number"
            value=""
          />
          <small
            class="error"
          />
        </div>
        <div
          class="formGroup"
        >
          <input
            class="input"
            name="address"
            placeholder="Endereço Cliente"
            type="text"
            value=""
          />
          <small
            class="error"
          />
        </div>
        <div
          class="formGroup"
        >
          <button>
            BUSCAR
          </button>
        </div>
        <div
          class="geometry"
        >
          <input
            class="input"
            disabled=""
            name="lat"
            placeholder="Latitude"
            type="number"
            value=""
          />
          <input
            class="input"
            disabled=""
            name="lng"
            placeholder="Longitude"
            type="number"
            value=""
          />
          <small
            class="error"
          />
        </div>
        <button
          class="button primary"
          type="button"
        >
          CADASTRAR CLIENTE
        </button>
      </div>
      <div
        class="form spacing"
      >
        <button
          class="button danger"
          type="button"
        >
          RESETAR CADASTRO
        </button>
      </div>
    </div>
    <div
      class="content"
    >
      <div
        class="container"
      >
        <div
          class="leaflet-container leaflet-grab leaflet-touch-drag"
          style="position: relative;"
          tabindex="0"
        >
          <div
            class="leaflet-pane leaflet-map-pane"
            style="left: 0px; top: 0px;"
          >
            <div
              class="leaflet-pane leaflet-tile-pane"
            >
              <div
                class="leaflet-layer "
                style="z-index: 1;"
              >
                <div
                  class="leaflet-tile-container leaflet-zoom-animated"
                  style="z-index: 18; left: 0px; top: 0px;"
                >
                  <img
                    alt=""
                    class="leaflet-tile"
                    role="presentation"
                    src="https://cartodb-basemaps-a.global.ssl.fastly.net/light_all/12/1517/2323.png"
                    style="width: 256px; height: 256px; left: -250px; top: -151px;"
                  />
                </div>
              </div>
            </div>
            <div
              class="leaflet-pane leaflet-shadow-pane"
            />
            <div
              class="leaflet-pane leaflet-overlay-pane"
            />
            <div
              class="leaflet-pane leaflet-marker-pane"
            />
            <div
              class="leaflet-pane leaflet-tooltip-pane"
            />
            <div
              class="leaflet-pane leaflet-popup-pane"
            />
          </div>
          <div
            class="leaflet-control-container"
          >
            <div
              class="leaflet-top leaflet-left"
            >
              <div
                class="leaflet-control-zoom leaflet-bar leaflet-control"
              >
                <a
                  aria-label="Zoom in"
                  class="leaflet-control-zoom-in"
                  href="#"
                  role="button"
                  title="Zoom in"
                >
                  +
                </a>
                <a
                  aria-label="Zoom out"
                  class="leaflet-control-zoom-out"
                  href="#"
                  role="button"
                  title="Zoom out"
                >
                  −
                </a>
              </div>
            </div>
            <div
              class="leaflet-top leaflet-right"
            />
            <div
              class="leaflet-bottom leaflet-left"
            />
            <div
              class="leaflet-bottom leaflet-right"
            >
              <div
                class="leaflet-control-attribution leaflet-control"
              >
                <a
                  href="https://leafletjs.com"
                  title="A JS library for interactive maps"
                >
                  Leaflet
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
      <h4>
        Total de Clientes 
        0
        ; Peso Total: 
        0
         kg; Ticket Médio*: 0
      </h4>
      <table
        class="table"
      >
        <thead>
          <tr
            class="tr"
          >
            <th
              class="th"
            >
              Nome
            </th>
            <th
              class="th"
            >
              Rua
            </th>
            <th
              class="th"
            >
              Cidade
            </th>
            <th
              class="th"
            >
              País
            </th>
            <th
              class="th"
            >
              Peso
            </th>
            <th
              class="th"
            >
              Lat
            </th>
            <th
              class="th"
            >
              Lng
            </th>
            <th
              class="th"
            >
              Action
            </th>
          </tr>
        </thead>
        <tbody>
          <tr>
            <td>
              No results found!
            </td>
          </tr>
        </tbody>
      </table>
      <h4>
        *Peso Total/Total de Clientes
      </h4>
    </div>
  </div>
</div>
`;
